mozilla-mobile/android-components/components/browser/errorpages/src/main/res/values/strings.xml

```diff
@@ -36,6 +36,19 @@
       </ul>
     ]]></string>
 
+    <!-- The text shown inside the advanced button used to expand the advanced options. It's only shown when a website has an invalid SSL certificate. -->
+    <string name="mozac_browser_errorpages_security_bad_cert_advanced">Advanced…</string>
+    <!-- The advanced certificate information shown when a website sends has an invalid SSL certificate. It's only shown when a website has an invalid SSL certificate. -->
+    <string name="mozac_browser_errorpages_security_bad_cert_techInfo"><![CDATA[
+        <label>Someone could be trying to impersonate the site and you should not continue.</label>
+        <br><br>
+        <label>Websites prove their identity via certificates. Firefox does not trust <b>%1$s</b> because its certificate issuer is unknown, the certificate is self-signed, or the server is not sending the correct intermediate certificates.</label>
```

Comment:
@MozLando : I think you landed these strings, so reaching out to you here. Could you please udpate the localization comment and call out what the placeholder '%1$s' refers to here? We do that for each string  that contains a placeholder. Thank you

Reply:
Passing by note: Firefox should really not be hard coded in strings, you're making forking the repository really complicated.

Reply:
Fixed: https://github.com/mozilla-mobile/android-components/pull/5624